pkg/ccl/backupccl/restore_mid_schema_change_test.go

```diff
@@ -53,60 +51,53 @@ import (
 // tables or a cluster backup. Most backups contain a single table whose name
 // matches the backup name. If the backup is expected to contain several tables,
 // the table names will be backupName1, backupName2, ...
-func TestRestoreMidSchemaChange(t *testing.T) {
-	defer leaktest.AfterTest(t)()
-	defer log.Scope(t).Close(t)
-
-	skip.UnderRaceWithIssue(t, 56584)
-
+//
+//lint:ignore U1000 unused
```

Comment:
I would expect that these functions, being used by generated code, do not trigger the unused checker. Is that not the case?